docs(popover): web-first (#4853)
digdir/designsystem: apps/www/app/content/components/popover/en/code.mdx

</div>
```

### Polyfill
Popover uses [popover (mozilla.org)](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/popover).
This api is classified as [Baseline: Newly available (mozilla.org)](https://developer.mozilla.org/en-US/docs/Glossary/Baseline/Compatibility) as of April 2024, when Firefox was the last browser to add it.

> In some cases, you may find that users are locked to older browser versions for various reasons, and then it may be appropriate to add a [Popover-Polyfill (github.com)](https://github.com/oddbird/popover-polyfill) to ensure that popover works for everyone.

Reviewer: ```suggestion
In some cases, you may find that users are locked to older browser versions for various reasons, and you might want to add a [Popover-Polyfill (github.com)](https://github.com/oddbird/popover-polyfill) to ensure that popover works for everyone.
```

```

## CSS variables and data attributes
Sizes are controlled with [data-size](/en/fundamentals/code/sizes) and colors with [data-color](/en/fundamentals/code/colors). 
The component will inherit from the closest parent where these are set.
<CssVariables />

<CssAttributes />

## React

```tsx
import { Popover } from '@digdir/designsystemet-react';

// with context
<Popover.TriggerContext>
  <Popover.Trigger>Trigger</Popover.Trigger>
  <Popover>Content</Popover>
</Popover.TriggerContext>

// without context with own trigger
<Button popovertarget="my-popover">Trigger</Button>
<Popover id="my-popover">Content</Popover>
```
### React and `popovertarget`

> When you use popover without popover.TriggerContext, you connect the trigger and popover yourself.

Reviewer: ```suggestion
When you use popover without `TriggerContext`, you connect the trigger and popover yourself.
```
